operations/firmware/Updating_Firmware_m001.md

```diff
@@ -29,9 +29,6 @@ The following information is needed:
 
 Use one of the following commands to find the model name for the node type in use.
 
-- [Find HPE model name](#find-hpe-model-name)
-- [Find Gigabyte model name](#find-gigabyte-model-name)
```

Comment:
Is GB supposed to be removed here?

Reply:
Did not think links were needed for that small section so removed.